### ISSUE TITLE
FYI [JMLR MOSS]

### ISSUE THREAD

**Issue by choltz95** [Alanthink/banditpylib#8]
Could be a long-term prospect if you are interested: [http://www.jmlr.org/mloss/](http://www.jmlr.org/mloss/)

Inspiration: [https://perso.crans.org/besson/articles/SMPyBandits.pdf](https://perso.crans.org/besson/articles/SMPyBandits.pdf)

**Comment by Alanthink** [comment]
Good idea. BTW, I have invited you as one of the collaborators. Now you can push prs directly to the master branch. You can add me to the reviewer if you want me to review your pr.

**Comment by Alanthink** [comment]
I also added a rule to the master branch such that every pr needs at least one of our approvals.
